ArduCopter/stand_by.cpp

```diff
@@ -0,0 +1,22 @@
+#include "Copter.h"
+
+
+// Run standby functions at 100Hz
```

Comment:
The 100Hz here is bogus.  You could say, "expected to be run at 100Hz", but the code which actually implements that 100Hz is very much elsewhere.  There are many other modes which have similar comments in them - many wrong, all out-of-place :-)

Reply:
yes, I agree we should remove these comments unless the function has some limitation that means it must run at only this speed (i.e. to stop devs from later modifying it).

Reply:
I have changed it to "Run standby functions at approximately 100 Hz to limit maximum variable build up" to make it more clear what the intended rate and purpose is. I hope that is more appropriate.